Exercises/date/date.js

```diff
@@ -0,0 +1,27 @@
+//1. Make a function which will return passed date object to passed format. Supported formats:
+
+function formatDate(date, format) {
```

Comment:
Your function works incorrectly in some cases:
```js
formatDate(new Date(Date.now() + 1000 * 3600 * 24 * 30 * 5), 'dd-mm-yyyy');
```

You can try to simplify it by using `.replace()` function